README.md

```diff
@@ -18,6 +20,23 @@ You can try out the user-side of Calypso on [WordPress.com](https://wordpress.co
 4.	Execute `make run` or `make dashboard` (for a more visually-oriented interface) from the root directory of the repository.
 5.	Open `calypso.localhost:3000` in your browser.
 
+### Docker
+
+1. Download and install [Docker](https://docs.docker.com/engine/installation/)
+2. Add `127.0.0.1 calypso.localhost` to your local hosts file.
+2. Run `docker-compose up`
+
+#### Some useful commands
+
+- `docker-compose up -d`: Run the container in the background, you can use `docker-compose logs -f` to follow them on-demand
+- `docker-compose down`: delete the container
+- `docker-compose down -v`: delete the container and node_modules
+- `docker-compose run --service-ports --rm calypso make dashboard`
```

Comment:
You have to specify `--service-ports` with the `run` command... otherwise it won't open ports.